shell/browser/extensions/electron_extension_loader.cc

```diff
@@ -110,6 +110,9 @@ void ElectronExtensionLoader::FinishExtensionLoad(
   if (extension) {
     extension_registrar_.AddExtension(extension);
   }
+  ExtensionPrefs* extension_prefs = ExtensionPrefs::Get(browser_context_);
+  extension_prefs->OnExtensionInstalled(extension.get(), Extension::ENABLED,
```

Comment:
What is this needed for?

Reply:
There was somewhere a DCHECK in web_request_api.cc code requiring this.

Reply:
https://source.chromium.org/chromium/chromium/src/+/master:extensions/browser/api/web_request/web_request_api.cc;l=2658;drc=6e8b402a6231405b753919029c9027404325ea00?q=web_request_api&ss=chromium%2Fchromium%2Fsrc

Reply:
And that's because there is no `ExtensionService` in Electron.
https://source.chromium.org/chromium/chromium/src/+/master:chrome/browser/extensions/extension_service.cc;l=1705;drc=c1908ef394f04e64cb4cdb8cc49e0e3b7f1c7527;bpv=0;bpt=1